pages.ru/common/rm.md

```diff
@@ -0,0 +1,24 @@
+# rm
+
+> Удалить файлы или каталоги.
+> Больше информации: <https://www.gnu.org/software/coreutils/rm>.
+
+- Удалить файлы из определённых мест:
+
+`rm {{путь/до/файла}} {{путь/до/другого/файла}}`
+
+- Рекурсивно удалить каталог и все его подкаталоги:
+
+`rm -r {{путь/до/директории}}`
+
+- Принудительно удалить каталог без запроса подтверждения или отображения сообщений об ошибках:
+
+`rm -rf {{путь/до/директории}}`
```

Comment:
```suggestion
- Принудительно удалить каталог без запроса подтверждения или отображения сообщений об ошибках:

`rm -r -f {{путь/до/директории}}`
```

Optional fix (error source: original page): other reason: joining short options is unpredictable in a general case.

Reply:
[This](https://github.com/tldr-pages/tldr-json-schemas) project will be used to solve this issue.